test/test_profiler.py

```diff
@@ -99,6 +102,76 @@ def forward(self, x):
 
         torch._C._set_graph_executor_optimize(prev_opt)
 
+    def test_python(self):
+        """Checks that python side high level events are recorded.
+        """
```

Comment:
need more tests.

Reply:
Thanks. Add pickle/unpickle test, and multi-level inherit test.